File: Project Academy/2. ViewModel/java/app/src/main/java/com/dicoding/academies/ui/academy/AcademyViewModel.java
package com.dicoding.academies.ui.academy;

import androidx.lifecycle.ViewModel;

import com.dicoding.academies.data.CourseEntity;
import com.dicoding.academies.utils.DataDummy;

import java.util.List;

public class AcademyViewModel extends ViewModel {

    public List<CourseEntity> getCourses() {
        return DataDummy.generateDummyCourses();
    }
}

File: Project Academy/02. ViewModel/kotlin/app/src/main/java/com/dicoding/academies/ui/academy/AcademyViewModel.kt
package com.dicoding.academies.ui.academy

import androidx.lifecycle.ViewModel

import com.dicoding.academies.data.CourseEntity
import com.dicoding.academies.utils.DataDummy

class AcademyViewModel : ViewModel() {

    fun getCourses(): List<CourseEntity> = DataDummy.generateDummyCourses()
}

